apache/trafficcontrol
# TP: adds ability to perform a quick search, override default pagination size and clear column filters to servers table

## What does this PR (Pull Request) do?

- [x] This PR is related to #4824 

The following improvements were added to the servers table:
- quick search across all columns
- ability to customize page size (override the default of 100)
- ability to clear all column filters
- adds a tooltip to the column values

![image](https://user-images.githubusercontent.com/251272/89204334-28c25080-d573-11ea-9166-6ecdaedac8ac.png)

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Run the UI tests and ensure they pass
- Navigate to https://tp.domain.com/#!/servers and try the following:
1. perform a quick search and ensure the rows are filtered appropriately. refresh. ensure value is persistent. 
2. change the pagination page size and ensure the page size changes. refresh. ensure value is persistent.
3. add one or more column filters and then hit the `clear column filters`  menu item under `More` to ensure the column filters are removed. 

## The following criteria are ALL met by this PR
